BEIMA.DB.Schemas/device-template-schema.txt

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
Not a huge deal, but could we change the name of this file to be 'device-type-schema'? It took me longer to understand what template meant in this sense since aren't all schemas a template for future entries?

Reply:
Resolved